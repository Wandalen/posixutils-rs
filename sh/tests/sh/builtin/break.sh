for x in a b c
do
  echo $x
  break
done

for x in a b c
do
  echo $x
  break 1
done

for x in a b c
do
  echo $x
  break 10
done

for x in a b c
do
  echo $x
  for y in d e f
  do
    echo $y
    break
  done
done

for x in a b c
do
  echo $x
  for y in d e f
  do
    echo $y
    break 1
  done
done

for x in a b c
do
  echo $x
  for y in d e f
  do
    echo $y
    break 2
  done
done

while true
do
  echo 'inside while'
  break
done

until false
do
  echo 'inside until'
  break
done