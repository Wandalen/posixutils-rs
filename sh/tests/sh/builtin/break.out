a
a
a
a
d
b
d
c
d
a
d
b
d
c
d
a
d
inside while
inside until
